qm.container

```diff
@@ -20,7 +20,6 @@ TasksMax=50%
 
 [Container]
 AddCapability=all
-DropCapability=sys_resource
```

Comment:
Is this the dup thing you found?

Reply:
No, it is not, it prevent from running the ffi-tools nested inside qm

Reply:
How about instead of removing commenting these options and add a comment why this is commended?

```
# DropCapability if enabled prevent ffi-tools nested inside qm.
# DropCapability=sys_resource
```

This might help us in the future. ;-)

Reply:
OK, ack

One more thing there is an issued with CID permission
 cidfile is placed in /run/user/ inside node1 is readonly, int it?

Reply:
Dropping this will allow FFI Tools to surpass their defaults.